subo/util/log.go

```diff
@@ -20,31 +21,36 @@ func (p *PrintLogger) LogInfo(msg string)  { LogInfo(msg) }
 func (p *PrintLogger) LogStart(msg string) { LogStart(msg) }
 func (p *PrintLogger) LogDone(msg string)  { LogDone(msg) }
 func (p *PrintLogger) LogFail(msg string)  { LogFail(msg) }
+func (p *PrintLogger) LogWarn(msg string)  { LogWarn(msg) }
 
-// LogInfo logs information
-func LogInfo(msg string) {
+// Keeping it DRY
+func Log(msg string) {
```

Comment:
@cohix In Ruby I'd put this function in a `private` block since it's not supposed to be called from outside. I'm guessing here I'm doing the same thing by not adding `Log` to the `PrintLogger` struct above, is that correct? I want to make sure I'm doing it in the Go way :)

Reply:
Actually, you can make it private by just lowercasing it: `log(msg string)` :)